addons/big-save-button/userstyle.css

```
@@ -0,0 +1,13 @@
+.save-status_save-now_2shdk span {
```

Probably want to use `[class*=...]` here. [Learn more](https://scratchaddons.com/docs/develop/userstyles/best-practices/#avoid-targeting-hashed-class-names)

This should work:
```suggestion
[class*="save-status_save-now"] span {
```

---

Ah! Yes! I thought of this one and then forgot about it 🤦

Thanks :D